qt_dotgraph/src/qt_dotgraph/edge_item.py

```diff
@@ -50,13 +56,17 @@ def __init__(self, highlight_level, spline, label_center, label, from_node, to_n
 
         self._default_text_color = self._COLOR_BLACK
         self._default_color = self._COLOR_BLACK
-        self._brush = QBrush(self._default_color)
+        self._text_brush = QBrush(self._default_color)
+        self._shape_brush = QBrush(self._default_color)
+        if style in ['dashed', 'dotted']:
+            self._shape_brush = QBrush(Qt.transparent)
```

Comment:
Otherwise, the triangular shape at the end of the arrow has a dashed border but a black fill. What looks strange.